src/com/teamcautionrobotics/robot2018/Robot.java

```diff
@@ -87,6 +93,31 @@ public void teleopPeriodic() {
             spinSpeed = manipulator.getAxis(Axis.RIGHT_TRIGGER);
         }
         intake.spin(spinSpeed);
+
+        boolean towerRaiseStateButton = manipulator.getButton(Button.Y);
```

Comment:
This is a rather clunky name. Something more like `liftRaiseButton` might be better.